doc/classes/Transform3D.xml

```diff
@@ -181,8 +181,25 @@
 	</members>
 	<constants>
 		<constant name="IDENTITY" value="Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0)">
-			A transform with no translation, no rotation, and its scale being [code]1[/code]. Its [member basis] is equal to [constant Basis.IDENTITY].
-			When multiplied by another [Variant] such as [AABB] or another [Transform3D], no transformation occurs.
+			The identity [Transform3D]. This is a transform with no translation, no rotation, and not scaled. Its [member basis] is equal to [constant Basis.IDENTITY]. This also means that:
+			- Its [member Basis.x] points right ([constant Vector3.RIGHT]);
+			- Its [member Basis.y] points up ([constant Vector3.UP]);
+			- Its [member Basis.z] points back ([constant Vector3.BACK]).
+			[codeblock]
+			var transform = Transform3D.IDENTITY
+			var basis = transform.basis
+			print("| X | Y | Z | Origin")
+			print("| %s | %s | %s | %s" % [basis.x.x, basis.y.x, basis.z.x, transform.origin.x])
+			print("| %s | %s | %s | %s" % [basis.x.y, basis.y.y, basis.z.y, transform.origin.y])
+			print("| %s | %s | %s | %s" % [basis.x.z, basis.y.z, basis.z.z, transform.origin.z])
+			# Prints:
+			# | X | Y | Z | Origin
+			# | 1 | 0 | 0 | 0
+			# | 0 | 1 | 0 | 0
+			# | 0 | 0 | 1 | 0
+			[/codeblock]
+			When this constant is transformed (multiplied) by a [Vector3], an [AABB], or another [Transform3D], no transformation occurs.
```

Comment:
I believe this is the opposite (_"When a variant is transformed by this constant"_)? This current text was derived from the original, rather vague statement.

Reply:
I have already done the change as I feel confident about this.